.sauce/testcafe_cloud.yml

```
@@ -0,0 +1,39 @@
+apiVersion: v1alpha
+kind: testcafe
+showConsoleLog: true
+sauce:
+  region: us-west-1
+  concurrency: 1 # Controls how many suites are executed at the same time (sauce test env only).
```

The comment isn't accurate anymore.

```suggestion
  concurrency: 1
```

---

👌. removing.